src/test/datascience/.env

```
@@ -1,2 +1,2 @@
-ENV_VAR_TESTING_CI=HelloWorldEnvVariable
-PYTHONPATH=./dummyFolderForPythonPath
+VSCODE_JUPYTER_ENV_TEST_VAR1=FOO2
+VSCODE_JUPYTER_ENV_TEST_VAR2=BAR2
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)